### ISSUE TITLE
What are "sane values"?

### ISSUE THREAD

**Issue by janxkoci** [BharatKalluri/Gifup#56]
I just couldn't make this app to do anything other than complain. Whenever I want to convert part of video (or whole video) to gif, it just throws a message saying "check if all fields have sane values and try again".

I tried to change everything but the app doesn't even suggest what's wrong. Is my gif too long? Dimensions of output are wrong? Of input? What? I feel like I need a PhD in GIFmaking just to use this thing, very frustrating.

Sane defaults should be set automatically by the app and not required from the user - user input is for special stuff.

**Comment by janxkoci** [comment]
So after some time I got the idea to start the app from terminal. It was complaining about various stuff, but I noticed one particular issue - there is a **hardcoded** output folder called "Downloads" in my home and unless I create that folder, there is no output (and it throws the "sane values" message above).

Should be easy to fix.

Edit: note that in non-english locales we don't have "Downloads" folder created by default.
